versions.tf

```
@@ -6,5 +6,9 @@ terraform {
       source  = "hashicorp/aws"
       version = "> 4.8.0"
     }
+    archive = {
+      source  = "hashicorp/archive"
+      version = "> 2.0.0"
```

```suggestion
      version = ">= 2.4.0"
```
Could we use the latest version? (same as above)

---

see slack